### ISSUE TITLE
Bug Report: Unlocking new playable character message doesn't apper

### ISSUE THREAD

**Issue by Szymonkruk12** [FunkinCrew/Funkin#3813]
### Issue Checklist

- [X] I have properly named my issue
- [X] I have checked the Issues/Discussions pages to see if my issue has already been reported

### Platform

Itch.io (Downloadable Build) - Windows

### Browser

None

### Version

0.5.3

### Description (include any images, videos, errors, or crash logs)

Please fix this bug because it does  not make it seen

### Steps to Reproduce

1. Open the game
2. beat weekend 1
3. check freeplay

**Comment by Hundrec** [comment]
Can you provide more detail? Is this happening on a brand new save file, or a save where you already have Pico unlocked?

**Comment by Szymonkruk12** [comment]
brand new save file

**Comment by Hundrec** [comment]
So you went to Story Mode, played through all four songs of Weekend 1, finished Blazin', and still didn't see "You can now play as Pico"?

**Comment by Szymonkruk12** [comment]
yes even if i check freeplay it doesn't show that pico is unlock

**Comment by EliteMasterEric** [comment]
Haven't been able to reproduce this yet
